CONTRIBUTING.md

```diff
@@ -129,12 +129,12 @@ Checklist:
   specifically reports the removal of the struct rather than all its fields separately.
 - Add the outputs you expect your query to produce over your test case in
   a new file: `test_outputs/<query_name>.output.run`.
-- Add `<query_name>` to the list of queries tested by the `query_execution_tests!()`
-  macro near the bottom of `src/query.rs`.
+- Add `<query_name>` to the list of queries used by the `add_lints!()`
+  macro near the bottom of `src/query.rs`. It includes the query content and also creates
+  a new test named `<query_name>` that compares the output of running this query on `test_crates/`
```

Comment:
Add the word "function" ("... creates a new test function named ...") and reflow text to keep lines short.
```suggestion
- Add `<query_name>` to the list of queries used by the `add_lints!()` macro near the bottom
  of `src/query.rs`. It includes the query content and also creates a new test function
  named `<query_name>` that compares the output of running this query on `test_crates/`
```